queryfilter/base.py

```diff
@@ -46,6 +46,27 @@ def false_with_drop_none_else_raise(self, field_name):
             return False
         raise FilterOnNoneValue(field_name)
 
+    def get_query_value(self):
+        return self.filter_args["value"]
```

Comment:
hmmm ... unfortunately as I remember it's not always value 😅 It'll change according to type 🤔

Reply:
ouch got it